### PR TITLE
DAOS-11363 test: Apply threshold for SCM free space check in container/multiple_delete.py

### PR DESCRIPTION
**Pull request by shimizukko**
After 50 iterations of container create, IOR run, and container destroy,
SCM free space isn’t fully recovered (about 20MB left). If we wait for
several seconds, most of them would be recovered and the remaining
data would be about ~198KB, which is considered not a bug.

Apply threshold of 400KB to the SCM free space check.

Increase pool size to avoid out of space during IOR.

Test-tag: multi_container_delete
Skip-unit-tests: true
Skip-fault-injection-test: true

Required-githooks: true
Signed-off-by: Makito Kano <makito.kano@intel.com>

### CODE REVIEW

**Review by daltonbohning — APPROVED** [review]

**Review by dinghwah — APPROVED** [review]

File: src/tests/ftest/container/multiple_delete.py [hunk 1]
```diff
@@ -30,7 +34,8 @@ def test_multiple_container_delete(self):
 
         :avocado: tags=all,full_regression
         :avocado: tags=hw,large
-        :avocado: tags=container,multi_container_delete
+        :avocado: tags=container
+        :avocado: tags=multi_container_delete
```
Comment (phender) [inline src/tests/ftest/container/multiple_delete.py:38]: We should also add the test method tag:
```suggestion
        :avocado: tags=multi_container_delete,test_multiple_container_delete
```

**Review by phender — CHANGES_REQUESTED** [review]
It would be good to fix the pylint error while we're fixing this test:
```
        for loop in range(50):
            self.log.info("Create-Write-Destroy Iteration %d", loop)
```

File: src/tests/ftest/container/multiple_delete.py (hunk 1 above)
Comment (shimizukko) [reply src/tests/ftest/container/multiple_delete.py]: Done

**Review by dinghwah — APPROVED** [review]

File: src/tests/ftest/container/multiple_delete.py [hunk 2]
```diff
@@ -53,7 +53,7 @@ def test_multiple_container_delete(self):
             self.run_ior_with_pool(create_cont=False)
             self.container.destroy()
             scm_fs, ssd_fs = self.get_pool_space()
-            out.append("iter = {}, scm = {}, ssd = {}".format(i+1, scm_fs, ssd_fs))
+            out.append("iter = {}, scm = {}, ssd = {}".format(loop+1, scm_fs, ssd_fs))
```
Comment (daltonbohning) [inline src/tests/ftest/container/multiple_delete.py:56]: Only if you need to push again - This warning shows up silently in flake, but we should have spaces around binary operators
```suggestion
            out.append("iter = {}, scm = {}, ssd = {}".format(loop + 1, scm_fs, ssd_fs))
```

**Review by daltonbohning — DISMISSED** [review]
LGTM

File: src/tests/ftest/container/multiple_delete.py (hunk 2 above)
Comment (shimizukko) [reply src/tests/ftest/container/multiple_delete.py]: Fixed

**Review by daltonbohning — APPROVED** [review]

**Review by dinghwah — APPROVED** [review]

**Pull request merged**